config/requirements.txt

```diff
@@ -1,4 +1,5 @@
 beautifulsoup4==4.9.3
 requests==2.25.1
 openai
-python-dotenv
\ No newline at end of file
+python-dotenv
+windows-curses
```

Comment:
This makes the app not run in other environments. For example on my system I get:

```sh
ERROR: Could not find a version that satisfies the requirement windows-curses (from versions: none)
ERROR: No matching distribution found for windows-curses
```

Screenshot below.

![image](https://github.com/nicobrenner/commandjobs/assets/2296756/3c6672a2-d75d-4a3c-8e84-57508cda3937)


It seems like there are some solutions for this, so that the installation process will use different packages depending on the operating system in use (see here: https://stackoverflow.com/questions/29222269/is-there-a-way-to-have-a-conditional-requirements-txt-file-for-my-python-applica)

This needs to be resolved before the PR can be merged, as it breaks the application for current users